llvm/lib/Target/DirectX/DXILResource.cpp

```diff
@@ -253,35 +246,6 @@ void UAVResource::print(raw_ostream &OS) const {
   ResourceBase::print(OS, "U", "u");
 }
 
-// FIXME: Capture this in HLSL source. I would go do this right now, but I want
-// to get this in first so that I can make sure to capture all the extra
-// information we need to remove the source type string from here (See issue:
-// https://github.com/llvm/llvm-project/issues/57991).
```

Comment:
Does this PR fix this issue? I did not see it was linked to this PR, but I may not be looking in the right place.

Reply:
This comment was kind of stale. That issue only mentioned capturing shape and ROV, which are both captured in the HLSL source before this change (and indeed the issue is closed). The issue this PR fixes is the obvious extension of that one, https://github.com/llvm/llvm-project/issues/70686